cmake/projects/ceres-solver/hunter.cmake

```diff
@@ -0,0 +1,29 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+# List of versions here...
+hunter_add_version(
+    PACKAGE_NAME
+    ceres-solver
+    VERSION
+    "1.12.0"
+    URL
+    "https://github.com/hunter-packages/ceres-solver/archive/hunter.zip"
+    SHA1
+    8554129d92cdddaead42274a47322709f82943e0
+)
+
+# Probably more versions for real packages...
+
+# Pick a download scheme
+hunter_pick_scheme(DEFAULT url_sha1_cmake) # use scheme for cmake projects
```

Comment:
Is this package cacheable/relocatable? See details: https://github.com/ruslo/hunter/wiki/usr.adding.new.package#important